clicache/integration-test2/Config.cs.in

```diff
@@ -19,8 +19,18 @@
 
 public class Config
 {
-  static public string GeodeGfsh
+  public static string GeodeGfsh
   {
     get { return @"@Geode_gfsh_EXECUTABLE@"; }
   }
+
+  public static string CryptoImplPath
+  {
+    get { return @"/../../../cryptoimpl/@CMAKE_BUILD_TYPE@"; }
```

Comment:
NO! That is isn't how you do this in CMake. Setup a variable in CMake that is something like `CRYPTO_IMPL_PATH` and changes this source to `@"@CRYPTO_IMPL_PATH@"`.

Reply:
will have a go at that